Yastrebov Yurii/analyze.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
+const os = require('os');
+const newLine = os.EOL;
+const fileName = process.argv[2].substring(1);
+const file = fs.readFileSync(fileName, 'utf-8', err => {
+  if (err) console.error(err);
+});
+const rowArray = file.split(newLine).map(line =>+line.split('-')[1]);
```

Comment:
Судя по фалу с самой игрой, при логировании к каждой строке добавляется символ перевода строки. Т.е. при сплите в массив попадет пустой элемент и значение `games` будет некорректным.
Я бы добавил `.filter(Boolean)` в цепочку перед `map`.

Reply:
Я начинающий в JS. Не очень понял для чего фильтровать Boolean, если в строке будут отсутствовать такие данные. Строка состоит из примерно вот этого: "1-0"

Reply:
```suggestion
const rowArray = file.split(newLine)
  .filter(Boolean)
  .map(line =>+line.split('-')[1]);
```
`filter(Boolean)` это более короткий аналог 
`filter(line => line)`
Т.е. отсеиваем пустые строки.